Formula/man-db.rb

```diff
@@ -0,0 +1,44 @@
+class ManDb < Formula
+  desc "Modern, featureful implementation of the Unix man page system"
+  homepage "https://nongnu.org/man-db/"
+  url "https://download.savannah.nongnu.org/releases/man-db/man-db-2.8.5.tar.xz"
+  sha256 "b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1"
+
+  depends_on "libpipeline"
+
+  def install
+    ENV["CC"] = "#{ENV.cc} -arch x86_64"
+    system "./configure", "--prefix=#{prefix}",
+                          "--with-systemdtmpfilesdir=no",
+                          "--with-systemdsystemunitdir=no",
+                          "--disable-cache-owner",
+                          "--disable-setuid"
+    system "make", "CFLAGS='-Wl,-flat_namespace,-undefined,suppress'"
```

Comment:
How do these cflags differ from the default ones?

Reply:
I'm not sure. Heck, I don't even know what the default `CFLAGS` are. Which should I remove?

Reply:
Going by the sudden build failure, I'd say those flags qualify as necessary.

Updated to use your suggestion.